### ISSUE TITLE
New bounty system conflicting with old

### ISSUE THREAD

**Issue by codexGW** [A3Armory/A3A_A3Wasteland.Altis#3]
The new bounty system and other changes are conflicting with old bounty set up. Here the error.

```
23:17:44 "WASTELAND SERVER - Main Mission 2 waiting to run: mission_Smugglers"
23:17:47 Error in expression <_player] call fn_loadAccount;

[[_this, _data],
{
params ["_pVal", "_data"];
_pV>
23:17:47   Error position: <_data],
{
params ["_pVal", "_data"];
_pV>
23:17:47   Error Undefined variable in expression: _data
23:17:47 File mpmissions\__cur_mp.Altis\persistence\server\players\setupPlayerDB.sqf, line 55
23:17:47 "pvar_requestPlayerData: [4,B Alpha 4-2:1 (Towley) REMOTE,B Alpha 4-2:1 (Towley) REMOTE]"
23:17:47 Error in expression <issionNamespace setVariable [_pvarName, _data];
(owner _player) publicVariableCl>
23:17:47   Error position: <_data];
(owner _player) publicVariableCl>
23:17:47   Error Undefined variable in expression: _data
23:17:47 File mpmissions\__cur_mp.Altis\persistence\server\players\setupPlayerDB.sqf, line 62
```

Here's the original commit and me try to fix the first merge issues.
https://github.com/A3Armory/A3A_A3Wasteland.Altis/commit/8cc0c58f99628b9e20183fdae8b15ff005a8587c#diff-e4a18cef2bed98b7e92ecd8b56600661L36
https://github.com/A3Armory/A3A_A3Wasteland.Altis/commit/99727402b7790d223bc30bfe40a5f213962c62df#diff-8d1aedc35081dff1536b8e8fcfbb1610R10

**Comment by AgentRev** [comment]
There is no "old" bounty, what are you referring to?

**Comment by codexGW** [comment]
This one that was added to the fork a couple weeks back.
https://github.com/A3Armory/A3A_A3Wasteland.Altis/commit/8cc0c58f99628b9e20183fdae8b15ff005a8587c

**Comment by AgentRev** [comment]
What are you doing with your commits? The thing I implemented is for you, you're not supposed to remove it, you are supposed to remove what you did and let the new stuff do its job. It's like a shiny new car and you're smashing the windows because you don't know where the key is...

**Comment by codexGW** [comment]
The merge was insane. I didn't know you did anything about the bounty. I though it was your stuff for the new one. I think Tyler asked this before hand.

**Comment by AgentRev** [comment]
There is no "new" or "old". I wrote load/save routines for **your** system. All you gotta do is plug it in.

**Comment by codexGW** [comment]
Oh okay. News to me :)

**Comment by codexGW** [comment]
I thought it was new because and not compatible because of the main config bounty stuff having different variables.

**Comment by AgentRev** [comment]
I will implement yours and alter it to use these variables instead. I put them in to not forget them.

**Comment by codexGW** [comment]
Cool, thanks man. Good work too! Looks like a epic ordeal for such little time and you being busy with work. Once we get the goal for the month met I will be donating some money to the A3W fund and encouraging others aswell. I only have a little set aside for the month for ArmA stuff and will use that once I get the servers back into shape.

**Comment by codexGW** [comment]
All better.
https://github.com/A3Armory/A3A_A3Wasteland.Altis/commits/dev

**Comment by AgentRev** [comment]
The truth is, I was unemployed and out of school between Apr 28 and Jun 26. It's kinda hard to get an IT summer internship when not studying in IT anymore. I was not doing much besides working on A3W and playing other games. I procrastinated about working on parking and storage until Jun 18.

I swear, the garbage sock code almost gave me a brain aneurysm. Just a couple hours ago I was still shouting at my screen. micovery clearly had some difficulties understanding order of execution. I managed to get the markers showing on top of territories and towns, instead of beneath.
